Purchases/Public/RCPurchases.m

```diff
@@ -596,7 +596,9 @@ - (RCPurchaserInfo *)readPurchaserInfoFromCache {
         NSDictionary *infoDict = [NSJSONSerialization JSONObjectWithData:purchaserInfoData options:0 error:&jsonError];
         if (jsonError == nil && infoDict != nil) {
             RCPurchaserInfo *info = [[RCPurchaserInfo alloc] initWithData:infoDict];
-            return info;
```

Comment:
So this will invalidate caches for all users on the new version. Are there unexpected side effects of doing this?

Reply:
The side effect is that offline users will not have access. There will be treated as if they had no cache at all.  Not invalidating has the side effect that there will be missing stuff in the entitlement infos.

Reply:
Got it... I think that's acceptable.